src/store/participantsStore.js

```diff
@@ -249,6 +251,14 @@ const getters = {
 		return {}
 	},
 
+	getPhoneStatus: (state) => (callId) => {
+		return state.phones[callId]?.state
```

Comment:
It will return `{ callid: string, status: string }`, as signaling sends it in value, but Participant component is waiting for `string` only
Or something is changed?

Reply:
I thought you wanted to be fixed like that, I didn't know it return an object